server/src/main/java/org/apache/cassandra/sidecar/modules/CassandraOperationsModule.java

```diff
@@ -112,6 +115,28 @@ VertxRoute cassandraCompactionStatsRoute(RouteBuilder.Factory factory,
         return factory.buildRouteWithHandler(compactionStatsHandler);
     }
 
+    @POST  // Note: POST, not PUT, since we're sending data in the body
+    @Path(ApiEndpointsV1.COMPACTION_STOP_ROUTE)
+    @Operation(summary = "Stop compaction operation",
+            description = "Stops a compaction operation on the Cassandra node")
+    @APIResponse(description = "Compaction stop operation completed",
+            responseCode = "200",
+            content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = CompactionStopResponse.class)))
+    @APIResponse(responseCode = "500", description = "Internal server error")
```

Comment:
Ideally we should not see `500` response from handler. 500 means we did not foresee some error and handle it properly. I would suggest removing it from list of expected responses.